modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/css/main.scss

```diff
@@ -53,3 +53,7 @@
 		margin-bottom: 0.5rem;
 	}
 }
+
+.dropdown-menu__translation {
```

Comment:
I would say that our [CSS class naming guidelines](https://github.com/liferay/liferay-frontend-projects/blob/master/guidelines/css/how_to_name_the_css_classes.md) don't officially endorse this `__` style, but neither do they say _not_ to do it (and there are over 500 instances in liferay-portal), so this seems fine.

Reply:
`.analytics-dropdown-menu` would fit better? 🤔  I didn't know the CSS class naming guidelines, sorry!

Reply:
I don't think it matters. If the tests pass, we forward, and if anybody wants a name change, it is trivial to do that in a follow-up.